README.md

```diff
@@ -8,17 +8,17 @@ Make custom error types that:
 
 ## For ES6
 
-ES6 classes make it easy to define custom errors by subclassing `Error`. Subclassing `errorType.Error` adds a few extra helpers.
+ES6 classes make it easy to define custom errors by subclassing `Error`. Subclassing `OErrors.OError` adds a few extra helpers.
 
 ### Usage
 
 #### Throw an error directly
 
 ```js
-const errorType = require('overleaf-error-type')
+const OErrors = require('overleaf-error-type')
```

Comment:
Hmm, plural or singular? To me it should be singular because there's just 1 error that is exported (barring the legacy es5 method)

Reply:
It was true but not anymore since we'll export `OError`. Will update